e2e/03-intraledger-flow.e2e.spec.ts

```diff
@@ -138,14 +138,19 @@ describe("Username Payment Flow", () => {
     const suggestionInput = await $(
       selector(LL.SendBitcoinScreen.suggestionInput(), "TextView"),
     )
-    await suggestionInput.waitForDisplayed({ timeout })
-    await suggestionInput.click()
-    await suggestionInput.setValue("e2e test suggestion")
-    await clickButton(LL.AuthenticationScreen.skip())
-
-    // FIXME: this is a bug. we should not have to double tap here.
-    await browser.pause(1000)
-    await clickButton(LL.AuthenticationScreen.skip())
+
+    try {
+      await suggestionInput.waitForDisplayed({ timeout })
+      await suggestionInput.click()
+      await suggestionInput.setValue("e2e test suggestion")
+      await clickButton(LL.AuthenticationScreen.skip())
+
+      // FIXME: this is a bug. we should not have to double tap here.
+      await browser.pause(1000)
+      await clickButton(LL.AuthenticationScreen.skip())
+    } catch {
+      /* empty */
```

Comment:
maybe add a comment on why the catch is necessary or useful instead of just `empty`

Reply:
done